gradle/helpers/test-tasks.gradle

```diff
@@ -34,10 +33,7 @@ if (System.env.TRAVIS == 'true') {
         test.dependsOn cucumberOAuthTest
     }
 } else {
-    // Separates the test task into unit and cucumber test tasks
-    test {
-        exclude '**'
-    }
+    test.dependsOn unitTest
```

Comment:
This seems unnecessary, there's a `test.dependsOn unitTest` right after the if-else.